core/modules/system/system.install

```diff
@@ -183,6 +183,22 @@ function system_requirements($phase) {
     $requirements['database_charset'] = $charset_requirements;
   }
 
+  // Report jQuery and jQuery UI versions.
+  if ($phase == 'runtime') {
+    $jquery = backdrop_get_library('system', 'jquery');
+    $jquery_ui = backdrop_get_library('system', 'ui');
+    $requirements['jquery'] = array(
+      'title' => $t('jQuery'),
+      'severity' => REQUIREMENT_OK,
+      'value' => $jquery['version'],
+    );
+    $requirements['jquery_ui'] = array(
+      'title' => $t('jQuery UI'),
+      'severity' => REQUIREMENT_OK,
```

Comment:
That could be a separate feature perhaps, but that sort of detection could be difficult to implement and validate. If a module replaces jQuery like D7's jQuery Update did, it should do so via `hook_library_info_alter()` and reflect the new version in the info array. So overall seems like a great addition.